src/bindings/python/test.py

```diff
@@ -50,7 +51,43 @@
 
                 print("  %s %12s%12f" % (prefix, candidate['plate'], candidate['confidence']))
 
+finally:
+    if alpr:
+        alpr.unload()
+
+
+try:
```

Comment:
It will be up to Matt at OpenALPR to decide how he wants this to look - we're introducing a dependency on OpenCV and numpy so he may want to separate it.